api/v1/server/handlers/api-tokens/list.go

```diff
@@ -0,0 +1,32 @@
+package apitokens
+
+import (
+	"github.com/labstack/echo/v4"
+
+	"github.com/hatchet-dev/hatchet/api/v1/server/oas/gen"
+	"github.com/hatchet-dev/hatchet/api/v1/server/oas/transformers"
+	"github.com/hatchet-dev/hatchet/internal/repository/prisma/db"
+)
+
+func (a *APITokenService) ApiTokenList(ctx echo.Context, request gen.ApiTokenListRequestObject) (gen.ApiTokenListResponseObject, error) {
+	tenant := ctx.Get("tenant").(*db.TenantModel)
+
+	tokens, err := a.config.Repository.APIToken().ListAPITokensByTenant(tenant.ID)
+
+	if err != nil {
+		return nil, err
+	}
+
+	rows := make([]gen.APIToken, len(tokens))
+
+	for i := range tokens {
+		rows[i] = *transformers.ToAPIToken(&tokens[i])
+	}
+
+	// This is the only time the token is sent over the API
```

Comment:
Ah to clarify - the actual token isn't included in the response because `gen.ApiTokenCreate200JSONResponse` is different from `gen.ApiToken` which doesn't include the actual token string. We're not storing the token anywhere in the database, just the token id, so it wouldn't be possible for us to get the raw token anyway.